.github/workflows/ci.yml

```diff
@@ -82,7 +84,7 @@ jobs:
       - name: Setup mdBook
         uses: peaceiris/actions-mdbook@v2
         with:
-          mdbook-version: 'latest'
+          mdbook-version: "latest"
```

Comment:
Nit: single quotes are used in other places of the yaml. Why we need double quotes here?